### DISCUSSION TITLE
[Suggestion] RGB Controller Modes

### DISCUSSION THREAD

**Discussion by zalexville** [tsunamayo/Starship-EVO#4262] [Suggestions]
### Additional information:

The idea would be to allow an RGB controller to switch between different lighting configurations. For example, my starship is using Star Trek-based alert modes. This means a Yellow, Red, and Blue alert. It would be nice to have the ability to have RGB controllers in my prefabs be able to switch between the lighting modes. This is due to lights/glow blocks only taking one input and the fact that RGBs cannot be linked via OR gates for multiple lighting configurations. 

This could also be applied to te number sequencer as it uses a similar conguration setup (would make evevator building easier)

Red Alert Status:
![sevo-red-alert](https://user-images.githubusercontent.com/65626266/142215556-7a4da9d6-d42f-45b1-a322-750f0b78d8e1.jpg)

No Alert: 
![sevo-no-alert](https://user-images.githubusercontent.com/65626266/142215930-6a9fefa9-373f-48d5-bd88-8583ec1c72c3.jpg)

**Comment by tsunamayo** [comment]
@zalexville That corridor is looking smooth! Yes being able to override (or route) config is something planned in the future - it will be useful for plenty of other features.
